### DISCUSSION TITLE
telegraf.conf zweite openDTU

### DISCUSSION THREAD

**Discussion by danny1605** [tbnobody/OpenDTU#1210] [Q&A]
Hallo ich habe eine Frage bezüglich der telegraf.conf. Meine Frage ist: Wie ist es möglich weitere topics auszulesen und auch diese in influx zu schreiben. Die erste openDTU sendet per MQTT an Telegraf, aber wie muss ich jetzt die conf so schreiben das ich das zweite topic ( 2 openDTU ) aufnehmen kann ? Hier die bestehende telegraf.conf 

# Read metrics from MQTT topic(s)
[[inputs.mqtt_consumer]]
  servers = ["tcp://X.X.X.X1883"]
 
  ## Topics that will be subscribed to.
  topics = [
    "solar/+/status/+",
    "solar/+/0/+",
    "solar/+/1/+",
    "solar/+/2/+",
    "solar/+/3/+",
   "solar/+/4/+"
  ]

data_format = "value"
data_type = "float"
tagexclude = ["host","topic"]
[[inputs.mqtt_consumer.topic_parsing]]
  topic = "solar/+/+/+"
  tags = "_/serial/channel/field"
  [[processors.pivot]]
    tag_key = "field"
    value_key = "value"

# store it in influx
[[outputs.influxdb_v2]]
  urls = ["http://X.X.X.X:32768"]
  token = "TOKEN"
  organization = "MY-ORG"
  bucket = "opendtu"

**Comment by spueli99** [comment]
Hi,

Du könntest das topic der 2. OpenDTU auf solar1 umstellen.
Dann müsste folgendes funktionieren.

```
topics = [
  "solar/+/status/+",
  "solar/+/0/+",
  "solar/+/1/+",
  "solar/+/2/+",
  "solar/+/3/+",
  "solar/+/4/+",
  "solar1/+/status/+",
  "solar1/+/0/+",
  "solar1/+/1/+",
  "solar1/+/2/+",
  "solar1/+/3/+",
  "solar1/+/4/+"
]

data_format = "value"
data_type = "float"
tagexclude = ["host","topic"]

[[inputs.mqtt_consumer.topic_parsing]]
topic = "solar/+/+/+"
tags = "_/serial/channel/field"

[[inputs.mqtt_consumer.topic_parsing]]
topic = "solar1/+/+/+"
tags = "_/serial/channel/field"

[[processors.pivot]]
tag_key = "field"
value_key = "value"

```
